lightbulb/commands/base.py

```diff
@@ -172,6 +172,18 @@ class OptionLike:
     """
     autocomplete: bool = False
     """Whether the option should be autocompleted or not. This only affects slash commands."""
+    name_localizations: t.Mapping[t.Union[hikari.Locale, str], str] = dataclasses.field(default_factory=dict)
+    """
+    A set of name localizations for this option
+
+    .. versionadded:: 2.3.0
+    """
+    description_localizations: t.Mapping[t.Union[hikari.Locale, str], str] = dataclasses.field(default_factory=dict)
+    """
+    A set of description localizations for this option
```

Comment:
These docstrings should be changed as it is incorrect. `"A mapping of locale to ... localizations for this option"` would be more appropriate

Reply:
Fixed in 05b489b88d4a2d8dc10a91c1a74c531fe867aacc.

Reply:
You didn't fix these ones